### PR TITLE
blog for data modernization

### PR DESCRIPTION
**Pull request by Sterling463**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by Sterling463** [comment]
build me

**Comment by rackernexus** [comment]
Build 1050 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-c6b5350d6/). valid for 14 days

**Comment by catlook** [comment]
publish Friday 5/8

File: _posts/2020-05-08-rackspace-offers-data-modernization.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+transform how data is collected, processed, and delivered to your stakeholders. This might start with
+a proof-of-concept on your data sample to demonstrate the value of data-driven decisions to your business.
+If you are further in your journey, we can develop a prototype or minimum viable product (MVP) to see how
+to leverage data analytics in practice. Or you may be ready for a full production solution which includes:
+
+- **Data Platform Architecture and Design:** Rackspace designs and builds an enterprise grade production
+environment leveraging automation, ingestion best practices, data processing, storage patterns and cloud best practices.
+- **Data Integration:** Rackspace develops data pipelines to ingest, transform, and store data.
+- **Data Modeling:** Rackspace data engineers define data models and create schemas, data translations, and consumption pattern best practices.
+- **BI & Reporting:** Rackspace data engineers will build dashboards and real-time data refreshes as well as build and automate operational and business reporting, so you have complete and readily available insight into your data.
```
Comment (catlook) [inline _posts/2020-05-08-rackspace-offers-data-modernization.md:66]: cut the "will" out

**Pull request merged**